test/functional/Valhalla/testng.xml

```diff
@@ -28,12 +28,20 @@
 	<test name="ValueTypeTests">
 		<classes>
 			<class name="org.openj9.test.lworld.ValueTypeTests"/>
-			<class name="org.openj9.test.lworld.ValueTypeArrayTests"/>
 		</classes>
 	</test>
 	<test name="ValueTypeTestsJIT">
 		<classes>
 			<class name="org.openj9.test.lworld.ValueTypeTests"/>
+		</classes>
+	</test>
+	<test name="ValueTypeArrayTests">
```

Comment:
Do we need ValueTypeArrayTests and ValueTypeArrayTestsJIT? They have the same class - `org.openj9.test.lworld.ValueTypeArrayTests`

Reply:
> Yes, but it is the same class. Can we just reference one ValueTypeArrayTests in playlist? i.e., -testnames ValueTypeArrayTests

OK, I see what you mean. There is no need to have `ValueTypeArrayTestsJIT` (and `ValueTypeTestsJIT`) in this file. 
@tajila has just merged this PR. I will address this comment in another PR.

Reply:
> I will address this comment in another PR.

https://github.com/eclipse-openj9/openj9/pull/16307